### ISSUE TITLE
Sqflint does not recognize Windows style paths

### ISSUE THREAD

**Issue by ultranoobian** [SkaceKamen/vscode-sqflint#23]
This valid line of code:
`_addzone = [] execVM "persistent\persistent_stats_zones_add.sqf";`
will produce this warning when using Windows style path separators:
`File 'persistent\persistent_stats_zones_add.sqf' doesn't exists.`

To my understanding, this is not expected behavior, especially since most (if not all) the examples used on the BI Wiki use this format.

In addition, prepending with a `/` to `persistent\persistent_stats_zones_add.sqf` the  will remove the warning which, again, isn't the expected outcome.

**Comment by SkaceKamen** [comment]
Afaik that shouldn't be the case, the \ symbols should work. Maybe the \p is recognized as some special symbol.
I'll try to update the code to replace those symbols.

**Comment by ultranoobian** [comment]
Whenever the project I'm working on has referenced a file not in the folder, they had always used the `folder\file.ext` format, and it seems to work.

So I'm not sure that last statement is the case.
Example: https://github.com/ultranoobian/DUWS-R/blob/master/source/INIT.sqf#L10

**Comment by SkaceKamen** [comment]
Yes, that's the right way, I was just saying you're right and it's not just recommended, it's required :)
